csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json

```diff
@@ -9,31 +9,59 @@
     "entry": [
       {
         "content": {
-          "src": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2024/va-24-317-01.json",
+          "src": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2024/va-24-325-01.json",
           "type": "application/json"
         },
         "format": {
           "schema": "https://docs.oasis-open.org/csaf/csaf/v2.0/csaf_json_schema.json",
           "version": "2.0"
         },
-        "id": "VA-24-317-01",
+        "id": "VA-24-325-01",
         "link": [
           {
-            "href": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2024/va-24-317-01.json",
+            "href": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2024/va-24-325-01.json",
             "rel": "self"
```

Comment:
This should be **"ref": "self"**

Reply:
Done.

Reply:
[CSAF 2.0 7.1.15](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#7115-requirement-15-rolie-feed) indicates "rel" which I think matches the [RFC](https://www.rfc-editor.org/rfc/rfc8322.html) also.

Reply:
@mstrad it looks like Art is correct. Should be "rel"
https://docs.oasis-open.org/csaf/csaf/v2.0/csd02/csaf-v2.0-csd02.html#7113-requirement-13-changescsv